README.md

```diff
@@ -54,9 +47,45 @@ async function handleEvent(event) {
 }
 ```
 
+### Options
+
+This function was designed to work with [Worker Sites](TODO:link-marketing-page), but you can customize the behavior of this function by passing the following options into the second argument to `getAssetFromKV`.
+
+#### `ASSET_NAMESPACE`
+
+type: KV Namespace Binding (required if not using Wrangler --site feature)
```

Comment:
```suggestion
type: KV Namespace Binding
```

Reply:
Adding this here adds confusion IMO. I think we should clearly define what each member does and then put an \* next to the ones that Workers Sites will configure for you. I can PR to this branch to show what I mean if you'd find that helpful @ashleymichal

Reply:
hmm. it is confusing. interested in what that looks like yah.